src/containers/ConceptPage/EditConcept.jsx

```diff
@@ -24,6 +24,14 @@ const EditConcept = ({ conceptId, selectedLanguage, t, ...rest }) => {
     return null;
   }
 
+  const isNewlyCreated = concept => {
```

Comment:
Litt usikker om det er en ok måte å løse oppgaven på. Saken er at vi har ikke en direkte tilgang fra CreateConcept til EditConcept, så det kan ikke sendes en prop direkte.

Reply:
Det virker litt som et hack. Er det tilsvarende problematikk for opprettelse av bilder og lyder? Korleis er det i tilfelle løst der?

Reply:
Problemet gjelder bilder, lyder, forklaringer og artikler. For problemmet ligger i som sagt at det er ingen direktekobling mellom Create og Edit, det er bare en router som re-router fra `/new` til `/edit`  og nettsiden lastes på nytt. Skal sjekke om det går an å sjekke hvor brukeren kom fra, for da kan jeg kanskje sjekke om brukeren ble redirected fra `/new` og lage en case for det.